scripts/dr_beam.py

```diff
@@ -6,6 +6,7 @@
 import json
 import time
 import numpy
+import numpy as np
```

Comment:
I'd switch over to using `numpy.` to keep the style consistent with the rest of the code.